src/telliot/utils/config.py

```diff
@@ -0,0 +1,58 @@
+import enum
+import pathlib
+from typing import Union
+
+import yaml
+from pydantic import BaseModel
+from yaml import CDumper as Dumper
+from yaml import CLoader as Loader
+
+
+class LogLevel(str, enum.Enum):
```

Comment:
Why do we have to enumerate logging levels if Python's built-in loggers already let you use different logging levels like: `logging.error("ahhh")` or `logging.info("hello")`?

Reply:
The idea is for the configuration to allow the user to choose the level of logging that he/she would like to see in the output.

However, I am not thrilled about having to enumerate the possible values in the `class LogLevel(str, enum.Enum)` class.  I was just trying to manipulate pydantic to export/import from a string in the config file.  Definitely open to improvements/suggestions here.

Reply:
I'm going to go ahead and merge this change,  but feel free to improve on the enumeration in the future.